ui/src/containers/HtmlHints.svelte

```diff
@@ -103,8 +103,8 @@
   let htmlRules;
   const getSelectedHtmlRules = async () => {
     await promise.then((data) => {
-      userSession$.subscribe(async x => {
-        const res = await fetch(`${CONSTS.API}/api/config/htmlhintrulesbyrunid/${runId}`);
+      userSession$.subscribe(async user => {
```

Comment:
Since this is in a function that gets called for updates, do we need a subscription here or can we just use get() to get the latest value?

i.e.
```
import { get } from "svelte/store";
...
const user = get(userSession$);
const res = await fetch(`${CONSTS.API}/api/config/${user.apiKey}/htmlhintrulesbyrunid/${runId}`);
htmlRules = await res.json();
```